python_modules/dagster-externals/dagster_externals_tests/test_external_execution.py

```diff
@@ -27,6 +27,7 @@
     ExternalExecutionFileMessageReader,
 )
 from dagster._core.instance_for_test import instance_for_test
+from dagster_aws.externals import ExternalExecutionS3MessageReader
```

Comment:
Should these tests instead live in the dagster aws package?

Reply:
Punting on this for now since we haven't yet figured out where external-system-specific injectors/loaders and readers/writers should live.